subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -1101,6 +1101,13 @@ static int tls_mbedtls_init(struct tls_context *context, bool is_server)
 		return ret;
 	}
 
+	if (context->options.ciphersuites[0] != 0) {
```

Comment:
I think this could be submitted as a separate bugfix PR, IMO this could get in rightaway.

Reply:
OK, I can pull it out of here and make a separate PR.

Reply:
done -- I'll pull it out of this when the separate one merges